### PR TITLE
More granular Datadog timings around HQ operations

### PR DESCRIPTION
**Pull request by wpride**
To help us determine source of slowness

### CODE REVIEW

**Comment by dannyroberts** [comment]
@wpride looks like tests are failing; probably just spring dependency injection–related?

File: src/main/java/aspects/MetricsAspect.java [hunk 1]
```diff
@@ -61,14 +73,37 @@ public Object logRequest(ProceedingJoinPoint joinPoint) throws Throwable {
...
 
+    private String getBlockedTime() {
+        long blockedDuration = restoreFactory.getDownloadRestoreTimer().durationInMs() +
+                installService.getInstallTimer().durationInMs() +
+                submitService.getSubmitTimer().durationInMs();
+        return "blocked_total:" + blockedDuration;
+    }
+
+    private String getRestoreBlockedTime() {
+        return "restore_blocked:" + restoreFactory.getDownloadRestoreTimer().durationInMs();
```
Comment (dannyroberts) [inline src/main/java/aspects/MetricsAspect.java]: You don't actually want this tag name here anymore, right? (You have `"restore_blocked_time:"` above.)

File: src/main/java/aspects/MetricsAspect.java [hunk 2]
```diff
@@ -61,14 +73,37 @@ public Object logRequest(ProceedingJoinPoint joinPoint) throws Throwable {
...
+    }
+
+    private String getInstallBlockedTime() {
+        return "install_blocked" + installService.getInstallTimer().durationInMs();
```
Comment (dannyroberts) [inline src/main/java/aspects/MetricsAspect.java]: same as above except that if you did intend to keep this, I think you want a `:` after the tag.

File: src/main/java/aspects/MetricsAspect.java [hunk 3]
```diff
@@ -61,14 +73,37 @@ public Object logRequest(ProceedingJoinPoint joinPoint) throws Throwable {
...
+    }
+
+    private String getSubmitBlockedTime() {
+        return "submit_blocked" + submitService.getSubmitTimer().durationInMs();
```
Comment (dannyroberts) [inline src/main/java/aspects/MetricsAspect.java]: same

**Comment by dannyroberts** [comment]
Generally looks good, had some comments

**Comment by wpride** [comment]
@dannyroberts thanks, sorry for missing that double tagging... :shame:

**Pull request merged**